### DISCUSSION TITLE
If a customer is logged in , editing their details via the admin doesnt take effect in the session until they logout/in again.

### DISCUSSION THREAD

**Discussion by hayden-t** [opencart/opencart#13853] [General]
Im talking about the session customer data that is populated when a customer logs in. After login its never updated again from the db and misses out on any changes made via admin.
These old incorrect details then flow through the checkout and into the saved order.
I created this issue but it was close right away by daniel with no discussion: https://github.com/opencart/opencart/issues/13850

One idea would be to add a last modified column to the customer table, and this is updated by admin on edit, and in catalog during customer construct, if they last modified is newer than the data in the session, then the session is updated.

**Comment by mavelo-llc** [comment]
Why not Instruct the customer to sign out and back in? I personally wouldn't want an extra query to be executed frequently with little benefit, two queries when outdated. They may not high overhead queries, but if you're already helping the customer, tell them what to do 😂

**Reply by hayden-t** [reply]
Well in that case there should be a note in the admin for to notify users that they must tell the customer to do this. I have data synchronised from external sources so this is not practical.

**Reply by hayden-t** [reply]
Also with the number of queries that happen in the background on any given page load, one extra one is insignificant.
